Bluejay.asm

```diff
@@ -3068,21 +3068,21 @@ dshot_tlm_create_packet:
 	mov A, Ext_Telemetry_H
 	jnz dshot_tlm_ready
 
-	; Read commutation period
 	clr	IE_EA
-	mov	Tlm_Data_L, Comm_Period4x_L
+	mov	A, Comm_Period4x_L			; Read commutation period
 	mov	Tlm_Data_H, Comm_Period4x_H
 	setb	IE_EA
 
-	; Multiply period by 3/4 (1/2 + 1/4)
-	mov	A, Tlm_Data_L
+	; Calculate e-period (6 commutations) in microseconds
+	; Comm_Period * 6 * 0.5 = Comm_Period4x * 3/4 (1/2 + 1/4)
+	;mov	A, Tlm_Data_L
```

Comment:
No need to carry commented out code, please remove completely.

Reply:
You're right, removed it.